src/api/custom-resolvers/mutations/delete-application-by-reference-number/index.ts

```diff
@@ -1,4 +1,5 @@
-import { Application, Context } from '.keystone/types'; // eslint-disable-line
+import { Context } from '.keystone/types'; // eslint-disable-line
```

Comment:
Please remove inline ESlint rule.

Reply:
This is required or lint will throw an error. @ttbarnes

Reply:
Yes i'm not a fan of this, I think we will have to create our own custom Context type, which will probably be an epic job. 

Tech debt. Will create a ticket.